databases/week3/week3.sql

```diff
@@ -0,0 +1,129 @@
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values(3, 'spring rolls','vegetable','ballerup','2020-12-13 14:02:10','6', '50' ,'2020-12-12'); 
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values(4, 'burger', 'chicken', 'lyngby', '2020-05-06 11:10:12' ,'5','120','2020-05-06');
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values(5, 'sandwitch', 'chicken', 'bagsværd', '2020-10-05 10:02:15', '3', '80', '2020-10-05'); 
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values(6, 'rød grød med flød', 'egg', 'buddinge','2020-08-12 12:05:10' , '2' ,'50', '2020-08-12');
+ 
+ SELECT * FROM meal
+ WHERE id=1;
+
+ UPDATE meal
+ SET title = 'pizza', location='farum'
+ WHERE id=1;
+ SELECT * FROM meal;
+
+  DELETE FROM meal 
+  WHERE id=2;
+  SELECT * FROM meal;
+
+ -- reservation table
+ insert into  reservation(id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values(1, '4','1','2020-10-20', '12345678','alexander','abc@mail.com');
+ insert into  reservation(id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values(2, '6','1','2020-10-20', '12312345','mikell','xyz@mail.com');
+ insert into  reservation(id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values(3,'4', '1', '2020-10-21','32456178', 'catherine' ,'svd@mail.com');
+  
+  SELECT * FROM reservation
+  WHERE id=1;
+
+  UPDATE reservation
+  SET contact_name= 'john'
+  WHERE id=1;
+
+ DELETE FROM reservation
+ WHERE id=2;
+
+-- review table
+ insert into  review(id, title, description, meal_id,  stars, created_date) values(1,'pizza', 'good', 1, '4', '2020-10-20'); 
+ insert into  review(id, title, description, meal_id,  stars, created_date) values(2,'pasta', ' not good', 1, '1', '2020-10-20');
+ insert into  review(id, title, description, meal_id,  stars, created_date) values(3,'sandwitch', ' yummy', 1 , '5', '2020-10-21');
+ 
+ SELECT * FROM review
+ WHERE id =1;
+
+ UPDATE review
+ SET description = 'super good'
+ WHERE id= 1;
+
+ DELETE FROM review 
+ WHERE id=2;
+
+ -- functionality
+
+ -- Get meals that has a price smaller than a specific price fx 90
+ SELECT * FROM meal 
+ WHERE price < 90;
+
+ -- Get meals that still has available reservations
+ SELECT *,  number_of_guests, max_reservations 
+  FROM meal
+  JOIN reservation ON meal.id = reservation.meal_id
+ WHERE max_reservations > number_of_guests;
+
+ -- Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+ SELECT * FROM meal
+ WHERE title LIKE 'rød grød med%';
```

Comment:
you can also consider adding ´%´ at the beginning of the string as well.
LIKE '%rød grød med%' - this way starting and ending could be anything. I hope you understood the difference.